plugins/markdown/src/org/intellij/plugins/markdown/ui/preview/javafx/MarkdownJavaFxHtmlPanel.java

```diff
@@ -111,9 +112,12 @@ protected String prepareHtml(@NotNull String html) {
   @Override
   public void setCSS(@Nullable String inlineCss, String @NotNull ... fileUris) {
     PreviewStaticServer.getInstance().setInlineStyle(inlineCss);
-    myCssUris = inlineCss == null ? fileUris
+    PreviewStaticServer.getInstance().setStyleOverrides(StyleOverridesKt.createStyleOverrides());
+    String[] baseStyles =
+      ArrayUtil.mergeArrays(fileUris, PreviewStaticServer.getStyleUrl(PreviewStaticServer.OVERRIDES_CSS_FILENAME));
```

Comment:
So, am I right that CSS will use last definition of color for selector even without `!important`?